sites/next.skeleton.dev/src/pages/docs/[...slug]/index.astro

```diff
@@ -15,23 +14,28 @@ export const getStaticPaths = (async () => {
 	}));
 }) satisfies GetStaticPaths;
 
+async function getComponentMeta(component: string) {
+	const meta = await getEntryBySlug('docs', `components/${component}/meta`);
```

Comment:
My previous version of the code did not hardcode `components`. You seem to be reverting this here. We should prepare for a future in which more than just Component section uses the split meta/framework structure for MDX files.

Reply:
You're totally right, my bad, I've removed the entire function and wrote a regex that just grabs the meta for the current route, this is indeed way more generic and will help when grabbing meta for future uses.